misc-utils/uuidd.c

```diff
@@ -515,9 +515,8 @@ static void server_loop(const char *socket_path, const char *pidfile_path,
 				       str, num - 1);
 			}
 			memcpy(reply_buf, uu, sizeof(uu));
-			reply_len = sizeof(uu);
 			memcpy(reply_buf + reply_len, &num, sizeof(num));
-			reply_len += sizeof(num);
```

Comment:
This does not seem correct. The second memcpy() uses still old offset from previous daemon call to write to reply_buf.

Reply:
You are absolutely right. Fixed in f84b3cd5ecdf0a75a87bb9745283a132c662e739.